browser/resources/settings/brave_appearance_page/sidebar.html

```diff
@@ -1,7 +1,11 @@
 <style include="settings-shared iron-flex">
+  .settings-box.bottom-border {
+    border-top: none;
+    border-bottom: var(--cr-separator-line);
+  }
 </style>
 <template is="dom-if" if="[[isSidebarFeatureEnabled_]]">
-  <div class="settings-box">
+  <div class="settings-box bottom-border">
```

Comment:
I think you can do it even easier than this - you might be able to (at the bottom, under the closing `div`, just put the following:
```
<div class="hr"></div>
```

Reply:
Actually - nevermind; this wouldn't work because it would need to be AFTER the template. This change looks good- consistent with the other styles on settings page 😄 👍 (ex: `settings-toggle-button`)